cmake/modules/RootMacros.cmake

```diff
@@ -915,9 +915,7 @@ function(ROOT_LINKER_LIBRARY library)
     target_link_libraries(${library} PUBLIC ${ARG_LIBRARIES} ${ARG_DEPENDENCIES})
   endif()
 
-  if(DEFINED CMAKE_CXX_STANDARD)
-    target_compile_features(${library} INTERFACE cxx_std_${CMAKE_CXX_STANDARD})
-  endif()
+  set_target_properties(${library} PROPERTIES CXX_STANDARD ${CMAKE_CXX_STANDARD})
```

Comment:
That change should be announced in the release notes. I think it's fine to remove the old one; I don't see why we would want to add the new call to `set_target_properties`, because according to [the doc](https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html)
> This property is initialized by the value of the [CMAKE_CXX_STANDARD](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD) variable if it is set when a target is created.

Reply:
Ok, let's not add the new line. But why would you announce this change in the release notes? The behavior is not changed by removing this, because the C++ standard is taken form `CMAKE_CXX_STANDARD` also by default?